design/pool-configuration.md

```diff
@@ -107,7 +107,7 @@ On top of that, Calico suggests to deploy MetalLB without the speakers to levera
 
 Currently, the options to make sure a given service is using a specific IP from a specific pool are:
 
-* using the `metallb.universe.tf/address-pool` annotation on the service
+* using the `metallb.io/address-pool` annotation on the service
```

Comment:
Only one nit: we keep design proposals as they were when they were discussed, regardless of the evolution of the project. So I'd ask to remove this change here.